app/scripts/services/membership/roles.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+angular
+  .module('openshiftConsole')
+  .factory('RolesService', function($q, DataService) {
+
+    var getClusterRoles = function() {
+      return DataService.get('clusterroles', null, {});
```

Comment:
remove this, getRoles, and getAllRoles completely, i do not want to see null'ed GETs